src/xrpld/app/tx/detail/Transactor.cpp

```diff
@@ -136,6 +144,23 @@ preflight1(PreflightContext const& ctx)
         ctx.tx.isFieldPresent(sfAccountTxnID))
         return temINVALID;
 
+    if (ctx.tx.isFlag(tfInnerBatchTxn) && !ctx.rules.enabled(featureBatch))
+    {
+        // The following transaction types dont have a tfUniversalMask flag
+        // check. Therefore these transactions need to be tesSUCCESS when the
+        // flag is set and the amendment is not enabled.
+        if (ctx.tx.getTxnType() != ttSIGNER_LIST_SET &&
+            ctx.tx.getTxnType() != ttCREDENTIAL_ACCEPT &&
+            ctx.tx.getTxnType() != ttCREDENTIAL_CREATE &&
+            ctx.tx.getTxnType() != ttCREDENTIAL_DELETE)
+            return temINVALID_FLAG;
+    }
```

Comment:
This checks are added in `fixInvalidTxFlags`, which goes into effect in a few hours, so they're therefore not needed here.

Reply:
fixed: https://github.com/XRPLF/rippled/pull/5060/commits/aa883e13a3319a56e4e7e498980cc01488ffe13b